backend/src/schema/types/type/User.gql

```diff
@@ -3,20 +3,16 @@ type User {
   actorId: String
   name: String
   email: String!
-  slug: String
-  password: String!
+  slug: String!
   avatar: String
   coverImg: String
-  avatarUpload: Upload
   deleted: Boolean
   disabled: Boolean
   disabledBy: User @relation(name: "DISABLED", direction: "IN")
   role: UserGroup
   publicKey: String
-  privateKey: String
-  
```

Comment:
this is why you removed the tests that @Tirokk added to check that we can't ask for a user's `privateKey`
meaning if it is removed from the type, we are unable to ask for it.

Reply:
Exactly, it is absolutely silly to declare it on the `User` type and then to make sure it's never exposed. Only because of `neo4j-graphql-js` it was there.

Reply:
great! another really nice side effect to not using it in this use case :smile_cat: